projects/plugins/boost/app/modules/cache/pre-wordpress/Boost_Cache.php

```diff
@@ -151,7 +151,7 @@ protected function delete_cache_for_front_page() {
 				$this->delete_cache_for_post( get_post( $posts_page_id ) );
 			}
 		} else {
-			$this->storage->invalidate_home_page( Boost_Cache_Utils::normalize_request_uri( home_url() ) );
+			$this->storage->invalidate( home_url(), Boost_Cache_Utils::JBCACHE_FILES );
```

Comment:
I think we don't need to JBCACHE prefix a constant that's already namespaced to a class